packages/pelagos/src/listInput/ListEntries.js

```diff
@@ -70,8 +94,18 @@ const ListEntries = ({
 							key={itemKey}
 							as="li"
 							className={`ListEntries__item${itemKey === highlightKey ? ' ListEntries__item--highlight' : ''}`}
-							data-testid="list-item">
-							{cloneElement(element, {className})}
+							tabIndex={reorderable ? 0 : undefined}
+							aria-describedby={reorderable ? operationId : undefined}
+							data-testid="list-item"
+							data-index={i}>
+							{reorderable ? (
+								<div className={`ListEntries__entry${reorderable ? ' draggable' : ''}`}>
```

Comment:
Missed this one: condition is redundant as `reorderable` is always true in this block.